examples/website/textures/app.js

```diff
@@ -35,21 +35,21 @@ export default class App extends PureComponent {
   constructor(props) {
     super(props);
 
-    const canvas = this.setupCanvas();
-    const gl = canvas.getContext('webgl');
-    const program = new Program(gl, {vs, fs});
-
     this.state = {
-      canvas,
-      gl,
-      program: program.handle
+      canvas: null,
+      gl: null,
+      program: null
     };
   }
 
   componentDidMount() {
-    const {gl} = this.state;
+    const canvas = this.setupCanvas();
```

Comment:
Nit: I'd put this in a helper function `initializeWebGL` - makes the code a bit more self documenting.

Reply:
Renamed